src/collections/_documentation/platforms/native/index.md

```diff
@@ -396,14 +402,14 @@ The Native SDK can use different backends that are responsible for capturing
 crashes. The backend is configured at build-time, using the `SENTRY_BACKEND`
 CMake option.
 
-The `crashpad` backend is used by default on Windows and macOS, whereas Linux
-and Android use the `inproc` in-process backend by default.
+The available options are `crashpad`, which is the default on Windows and macOS,
```

Comment:
_nit_: What if we make this a list? It might read easier when scanning through the doc.